src/views/settings/SettingsNotifications.tsx

```diff
@@ -176,10 +176,12 @@ const SettingsNotifications: Screen<"SettingsNotifications"> = ({
                 entering={anim2Papillon(FadeInDown).delay(70 * index)}
                 trailing={
                   <Switch
-                    trackColor={{
-                      false: colors.border,
-                      true: colors.primary,
-                    }}
+                    trackColor={
+                      {
+                        false: theme.colors.border, true: theme.colors.primary
+                      }
+                    }
+                    thumbColor={theme.dark ? theme.colors.text : theme.colors.background}
```

Comment:
J'te fais la suggestion de code que @Gabriel29306 demande :


```suggestion
                    trackColor={
                      {
                        false: colors.border,
                        true: theme.colors.primary
                      }
                    }
                    thumbColor={theme.dark ? colors.text : colors.background}
```